app/services/backfill_qualification_public_id.rb

```diff
@@ -0,0 +1,26 @@
+class BackfillQualificationPublicId
+  def initialize(qualification)
+    @qualification = qualification
+  end
+
+  def call
+    if @qualification.constituent_grades.present?
```

Comment:
Ah this is wrong, since triple science GCSEs will be treated differently.

We should set the top level public_id on triple science ones since the grades for that subject are just glued together in the API like `AAB`